Box changes

Probe1 opened the pull request:
**Please see the comments for images.**
### Overhauls
- Significant alteration to botany back room and the adjacent maint to accommodate it. It was expanded by 1x9 tiles, the grinder was moved into the back room, and a paperbin, pen, labeler, folder, chair, and 3 tables were added.  The sink was removed to stop farm bots from pathing towards it and getting stuck in a loop. This has been requested for 6 months. Someone requested it in the thread and reminded me.  https://github.com/d3athrow/vgstation13/pull/6792#issuecomment-160723876
- Overhauls Research Botany at the Research Outpost.  The room is significantly changed in layout.  The two xenobotany machines are added.  A smartfridge, desk, and office supplies are added.  Connectors and a vent to space are added so the trays can remove dangerous gasses.  A hatchet and wrench are added to the plant crate.  Two plant analyzers are added.  https://github.com/d3athrow/vgstation13/pull/6792#issuecomment-160810513
### Tweaks
- Resolves my own issue https://github.com/d3athrow/vgstation13/issues/6542 by adding a new area, and a new icon for the area, and a new APC for the area known as permabrig.  I named it Permanent Confinement to keep in line with the lingo.  This will drastically reduce the power drain on the prison wing APC and bring the APCs in the brig in line with the rest of the station in terms of power drain.  I placed the APC on top of an existing power line because it was the closest free place available, allowing security to see it without entering perma.  The new area is immune to radiation storms, as it was before with the old area.  https://github.com/d3athrow/vgstation13/pull/6792#issuecomment-160544873
- I resolved https://github.com/d3athrow/vgstation13/issues/6681 by a slight modification of the existing pipe line and removing the other sort junction, supplanting it with a new one. https://github.com/d3athrow/vgstation13/pull/6792#issuecomment-160543498
- Fixes https://github.com/d3athrow/vgstation13/issues/6559 
- Adds a mailman outfit to an undisclosed location for players to find.  One guy can arrpee as a hounddog mail worker named Jjenjr.
- Adds two pairs of magboots to the research outposts Expedition Preparation Room.  
- Removes a random errant camera in maint above botany backroom that was labeled as a solar panel camera.  Someones unnoticed whoops.  This is why you don't have your cursor set to place mode when you're not placing things!
- Changes a reinforced table in the bar next to the booze-o-mat to a regular table so redecoration isn't as laborious.  Request came from the thread.

Probe1 commented:
I was merely pretending to be retarded in my first commit!  This should resolve it.

Probe1 commented:
new area, new icon for perma.
![image](https://cloud.githubusercontent.com/assets/8550416/11465874/d6663096-970a-11e5-9210-c3f18717e90a.png)

Probe1 commented:
Mailman outfit for anon in the thread.  Seemed like a nice idea.  

![image](https://cloud.githubusercontent.com/assets/8550416/11466282/c90db0ce-970d-11e5-9662-d670876f533e.png)

Probe1 commented:
![image](https://cloud.githubusercontent.com/assets/8550416/11467730/187bde74-9718-11e5-904c-c630b0e251af.png)

Significant alteration to botany back room and the adjacent maint to accommodate it. It was expanded by 1x9 tiles, the grinder was moved into the back room, and a paperbin, pen, folder, chair, and 3 tables were added. This has been requested for 6 months. Someone requested it in the thread and reminded me.

@Bauds @TermedClepe please look over it and see what you think

Probe1 commented:
Added the new sec_perma area to the exception list for rad storms to keep those poor prisoners from dying.  Thank you for the reminder @ArthurDentist and the help on the other thing.

TermedClepe commented:
Remove the sink from the botany back room. It makes botany bots stuck on the airlock trying to refill water.

Other than that looks good, everything in place.

ihadtoregisterforthis commented:
It would look nicer if the APC wasn't inside the prison area. Perhaps moving it between the double airlocks?

Probe1 commented:
@ihadtoregisterforthis It's gotta be its just gotta!  No really it's a balance thing. Forcing a player to enter to use its air alarm and APC and possibly run into a prisoner creates interesting situations, more than not.

9600bauds commented:
>botany backroom

!!!YES!!!

Two things, however, I'm no atmosman but I think you might want to move the vent out from under the table. Second being switch the lysis-isolator and bioballistic delivery system around becuase it triggers my autism pls ok ty.
Otherwise, fantastic, absolutely fantastic

Probe1 commented:
ravioli ravioli give me the formuoli

	Removes the sink from botany, switches the lysis and bioballistic machines, and modifies the waste line and scrubber to not be so spaghetti.

9600bauds commented:
never forgetti

Probe1 commented:
I don't want to admit it but I rather autistically positioned that labeler and paper bin so it looked nice rather than the usual shit heap of items we find on tables.  Unless @Kurfursten or anyone sees anything wrong I think the PR is complete.

Kurfursten commented:
Pipes look good. I would move the table and chemmaster a square right. That frees up more space near the door, which is useful when you're moving exotic crates into the back room - especially when you have several of them.

Probe1 commented:
Works for me.  I put it that way because I believe having that hole in the tables made the room feel more open than it was, giving players the illusion of space even if they wouldn't use it.  But we can nudge them.

I added magboots to the research outposts expedition room which has been requested forever and universally wanted.  

![image](https://cloud.githubusercontent.com/assets/8550416/11483611/4bf8f1ce-9777-11e5-82be-1ccaa251677c.png)

9600bauds commented:
If you're moving the chemmaster, flip it and the grinder around for muh table flush

Probe1 commented:
That's what I did yes.

Probe1 commented:
I significantly altered xenobotany at the research outpost to accommodate the xenobiology machines and make it a functional addition instead of a cupboard people go to snack on wild mushrooms and ignore.

Probe1 commented:
Research Botany/ Xenobot

![west_object_48_2015-11-30_19-43-48](https://cloud.githubusercontent.com/assets/8550416/11489174/ddaf3308-979a-11e5-9bc0-e019b8d65c27.png)

Kurfursten commented:
Why is there a furry paw in tiles?

Probe1 commented:
 >furry paws
well that's ruined for me now.  The map tools are missing the correct tile sprites.  I made an arrow.  Or  paws.  It adds texture to the room.

Kurfursten commented:
It's not missing the correct sprite, talk to me on irc

TermedClepe commented:
I'm legitimately mad at science getting xenobotany with chem dispenser next to it.

Why would I want to play a botanist anymore when I can do the exact same thing just better at science?

Kurfursten commented:
@TermedClepe He was talked down from giving them 16 trays, so at least botany still has
* Easier mail from Cargo for exotics and other stuff
* 2.5x more trays

TermedClepe commented:
@Kurfursten
Tray count is a joke since you can have infinite trays from grass.

Science botany has a fucking chem dispenser right in it.

Probe1 commented:
@TermedClepe We've had a research botanist job for a long time.  They got a dogshit cupboard without xenobotany machines.  Their job was pointless.

I don't like the job.  I think it shouldn't exist.  But it does and people have been begging for the room to be re done.  If you want it removed, make a discussion issue, get some people talking, I'll be right there supporting it.  But as long as it's still in the game then it needs to have parity with the botanist.

Kurfursten commented:
I guess. But by the same token, radium is really easy to get as a normal botanist.

In Botany2, a chem dispenser isn't really a big deal since it's all just dice rolls to get stat ups anyway, but it will be ***huge*** when Botany3 hits.

@Probe1 Might be worth talking about killing Research Botanist in the Botany3 PR. Why don't you do some polling to see how much people even want to keep it?

Probe1 commented:
More people wanted to keep it than remove it which is why I just spent 3 hours mapping.

Probe1 commented:
And to fight for my own changes:  The only major additions here are the xenobotany machines (the *point* of being a research botanist is to use them), two free exotic seeds, two additional trays (which is still far behind Baystation who have 16) and a lot better layout of the room.

TermedClepe commented:
>free exotic seeds
I can't be more mad.

Probe1 commented:
I will remove them if it's a sticking point.  I didn't think 2 was egregious when a crate comes with 6-8 and often times ends up with no good exotic seeds.

TermedClepe commented:
It's ok you can leave it.

The power creep will become clearer once botany overhaul gets merged. I'll make a new issue after that as this is basically a derailment about the bigger problem that is research botany.

Probe1 commented:
Too late they're gooooooone.

TermedClepe commented:
I think science could really use an increased job cap at round start since it has so many different things to do.

Kurfursten commented:
Science could do with some role-splitting to be perfectly honest. I've seen a shift where THREE Xenoarcheologists joined. Best of all? The mechanic decided to abandon his workshop to play xenoarch. Meaning two xenoarchs had no access to a suit to do their job.

Probe1 commented:
I'm sure they were comfortable in their NASA Voidsuits.

Probe1 commented on code/game/area/Space Station 13 areas.dm:
```diff
@@ -2079,14 +2083,14 @@ proc/process_adminbus_teleport_locs()
 		if(Obj:client)
 			mysound.status = SOUND_UPDATE
-			Obj << mysound
+			Obj << mysound
```
What are all these?  Did the map cleaner decide to fuck with me?

PJB3005 replied:
Hrm.

It might be as a result of the `to_chat()` refactor in some way?

Intigracy commented on html/changelogs/example.yml:
```diff
@@ -12,8 +12,8 @@
-#   rscdel (general adding of nice things)
-#   rscadd (general deleting of nice things)
+#   rscdel (general deleting of nice things)
+#   rscadd (general adding of nice things)
```
fag, leave our memes alone

Probe1 replied:
no i must fix things

Probe1 commented:
The dumb paw design in research bot was removed by the way, didn't explicitly state it.

9600bauds commented:
What the christ is this talk about a designated yiffing zone

Probe1 commented:
we should rename the tox burn chamber as the yiff zone.

9600bauds commented:
No really what the dickens did I miss

Probe1 commented:
Nothing.   ``` dir ``` confused me and I mistakenly thought there wasn't the correct tile paint option.  So I did a quick little arrow design.  Kurfurst pointed out it looked like a paw, then pointed out that ``` dir ``` went up to 10 rather than 8 as I believed.  I corrected the tile paint that was wrong and removed the pawrrow.

Probe1 commented:
Two minor fixes.

Fixes an errant research hall area in the research shuttle bay, causing fires to trigger alarm overlays in a safe area.  Also removes the weird ass tiles in the bar that merely LOOKED like wooden tiles, but were really metal tiles painted to appear as wood.  I don't even know man.

Probe1 commented:
ok i'm done no more.  I need to move on to a new map.

Probe1 commented:
Alright 24 hours have passed and the only substantive feedback I've received from posting it in OOC/thread is that @TermedClepe doesn't want Research Botanists to be a thing, which is a separate issue.  And @SirCrab reminding me to add detective manuals to all maps (and fix the code for them) *after* his PR to fix the console is merged, which is for a separate PR after his.

9600bauds commented:
merge now before confrictu
